gr-fec/grc/tpc_decoder_def_list.block.yml

```diff
@@ -65,9 +65,11 @@ templates:
         % if int(ndim)==0:
         self.${id} = ${id} = fec.tpc_decoder_make(${row_poly}, ${col_poly}, ${krow}, ${kcol}, ${bval}, ${qval}, ${max_iter}, ${decoder_type})
         % elif int(ndim)==1:
-        self.${id} = ${id} = list(map( (lambda a: fec.tpc_decoder_make(${row_poly}, ${col_poly}, ${krow}, ${kcol}, ${bval}, ${qval}, ${max_iter}, ${decoder_type})), range(0,${dim1}) ))
```

Comment:
There are many such issues with the yml in gr-fec that cause the passed objects not to be understood.  
Mostly fixed by adding list(map(...))  and some line continuation issues such as lambda\a